Extension/lib/content-script/assistant/js/adguard-rules-constructor.js

```diff
@@ -105,6 +94,21 @@ var AdguardRulesConstructorLib = (function (api) {
         return path.join(" > ");
     };
 
+    var getElementClassName = function (element) {
```

Comment:
1. Don't remove spaces If you wrap class in [class="class1 class2 class3"]
2. With enabled 'Block similar elements' option css selector matches all of elements that contain any of class in list, like:
```
<div class="class1 class2 bad.class3"></div>
Rule: #.class1, .class2, [class="bad.class3"]
```